tasks/host_volume.yml

```diff
@@ -7,3 +7,4 @@
     state: "{{ item['state'] | default('directory') }}"
     mode: "{{ item['mode'] | default('0755') }}"
   with_items: "{{ nomad_host_volumes }}"
+  ignore_errors: true
```

Comment:
Is there a reason this and the config in the client.hcl.j2 is added here?

Reply:
I removed it